### ISSUE TITLE
Update to Bevy 0.9

### ISSUE THREAD

**Issue by olegomon** [bonsairobo/smooth-bevy-cameras#25]
## Summary

As of today Bevy is 2 open issues behind the 0.9 milestone. See: https://github.com/bevyengine/bevy/milestone/7 and this crate should support it asap ;)

**Bevy Migration**
- remove deprecated API usage of `spawn_bundle()` and use `spawn()` instead
- remove deprecated API usage of `insert_bundle()` and use `insert()` instead

**Maintenance**
- update all crate dependencies

**Comment by olegomon** [comment]
@bonsairobo thank you for providing this crate. I really like it and it helps me a lot by building a simple sandbox application to learn all about shading etc. I want to switch to Bevy 0.9 soon so I prepared a PR for an Bevy 0.9 update: https://github.com/bonsairobo/smooth-bevy-cameras/pull/26

Feel free to comment if I missed something or if something could be improved :)

**Comment by cryscan** [comment]
Hi! When will there be a crate.io release?

**Comment by bakcxoj** [comment]
@bonsairobo Are you able to publish to crates.io?

**Comment by DSchroer** [comment]
:+1:  on the need for 0.9. I'm operating off the PR branch but would like to go back using an official release.

**Comment by bakcxoj** [comment]
Same. I'm having to do this:

```toml
[dependencies]
bevy = "0.9"
smooth-bevy-cameras = { version = "0.6.0", git = "https://github.com/bonsairobo/smooth-bevy-cameras" }
```

**Comment by bonsairobo** [comment]
I just published 0.6 to crates.io.
